### DISCUSSION TITLE
Backup / export / restore from the GUI

### DISCUSSION THREAD

**Discussion by gtothill** [jonaswinkler/paperless-ng#1269] [Feature Requests]
Firstly thanks for great software. It’s revolutionised how we deal with paper in our house.
Because various non technical members of the household use the system, it would be incredibly helpful to be able to set the options for a document export and initiate that export, from the Web GUI. I say this because although the command line option works very well, non technical users find it difficult to routinely navigate a terminal.
So for the admin to be able to setup export options and the target directory and the user to be able to initiate an export and a restore from the GUI would be incredibly helpful.
All the best,

Geoff.

**Comment by hendrikma** [comment]
Yeah, maybe there should be some sort of encrypted deduplicated backup-solution to manage from GUI like Borg-Backup!

**Comment by gtothill** [comment]
There is an excellent built in backup and restore mechanism that has app specific features and all I was suggesting was that for non technical users it would be good if that were possible to manage from the GUI, but thanks for pointing out that backup solutions for Linux exist.

________________________________

**Comment by obizbu** [comment]
Came here to request the same thing. I'm familiar enough with docker to generate the export from there, but would be nice to have a button in the settings that I could click and have it generate an export - either in the configured directory OR generated as a zip file (bonus for encryption) as stated above

**Comment by obizbu** [comment]
One other suggestion for this:

Allow PaperlessNg to be configured in such a way that every time a document is consumed, it writes the document + PNGs and associated metadata to a particular file. Currently, I run the export job every time I upload a few documents.
